src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -36,8 +37,31 @@
  * Checks that
  * <a href="https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#firstsentence">
  * Javadoc summary sentence</a> does not contain phrases that are not recommended to use.
- * Summaries that contain only the {@code {@inheritDoc}} tag are skipped.
- * Check also violate Javadoc that does not contain first sentence.
+ * Summaries that contain only the {@code {@inheritDoc}} tag are skipped for normal summary javadoc
+ * but are violations for inline summary javadoc. Check also violate Javadoc that does not contain
+ * first sentence. </p><p> According to
+ * <a href="https://docs.oracle.com/javase/10/docs/specs/doc-comment-spec.html#general-syntax">
```

Comment:
Please use Java 11 (the latest LTS release) for links. For example:
https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#general-syntax

Reply:
Done

Reply:
Sir, please restart Cirrus. 
Failure log
```
Failures
 - ant (exited 1) - ant not installed. An error occurred during installation:
 The remote server returned an error: (503) Server Unavailable. Service Unavailable
```